restapi/api.go

```diff
@@ -31,6 +31,7 @@ func Router(db *gorm.DB) (http.Handler, error) {
 		rest.GetHandler(r, "/health", api.health)
 		rest.GetHandler(r, "/stories", api.ReturnAllStories)
 		rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
+		rest.GetHandler(r, "/authors/countries", api.ReturnAllCountries)
```

Comment:
can we make it `authors/origin_countries` because over here countries refer to origin countries